pyvrp/_pyvrp.pyi

```diff
@@ -41,7 +41,7 @@ class Client:
         demand: int = 0,
         service_duration: int = 0,
         tw_early: int = 0,
-        tw_late: int = 0,
+        tw_late: int = ...,
```

Comment:
https://docs.astral.sh/ruff/rules/argument-default-in-stub/ - Ruff disagrees with using `np.iinfo(np.int32).max` here as the default.